Prog/AddNew/TOPA_match_notices_property.sas

```diff
@@ -0,0 +1,100 @@
+/**************************************************************************
+ Program:  TOPA_match_notices_property.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Elizabeth Burton
+ Created:  12/13/2022
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Match TOPA notices to the same properties
+ 
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( DHCD )
+%DCData_lib( MAR )
+%DCData_lib( RealProp )
+
+%File_info( data=PresCat.TOPA_SSL, printobs=5 )
+%File_info( data=PresCat.TOPA_realprop, printobs=5 ) /** some IDs don't have real_prop info**/
+%File_info( data=PresCat.TOPA_addresses, printobs=5 )
+%File_info( data=PresCat.TOPA_database, printobs=5 ) /** 1699 obs**/
+
+/** merge address_id ssl and real_prop info **/
+
+data Topa_address_ssl_realprop;
+  merge
+    PresCat.TOPA_SSL (keep=id address_id ssl in=in_ssl)
+    PresCat.TOPA_addresses (keep =id address casd_date offer_sale_date in=in_addresses);
```

Comment:
I took out the real property data set here, which I don't think we need for this step. And doing 3-way merges in a data step can lead to unpredictable results.